analyzer/src/main/kotlin/managers/utils/NuGetAllPackageData.kt

```diff
@@ -35,6 +35,7 @@ internal class NuGetAllPackageData(
         val resources: List<ServiceResource>
     )
 
+    // See https://docs.microsoft.com/en-us/nuget/api/service-index#resource.
```

Comment:
That raises the question to me, why don't we have those as KDoc comments so that they would appear in the generated docs?

Reply:
Because these are not really (user) docs in my view, but references for the programmer.